templates/UmbracoProject/.template.config/template.json

```diff
@@ -51,7 +51,7 @@
       "description": "The version of Umbraco.Cms to add as PackageReference.",
       "type": "parameter",
       "datatype": "string",
-      "defaultValue": "10.0.0-rc1",
+      "defaultValue": "*",
```

Comment:
This value will be replaced in the custom build target, so it doesn't really matter what value is used here: I've set it to `*` to still have a valid version value, but it could be left empty if you'd prefer that...